UnitsNet/QuantityFormatter.cs

```diff
@@ -13,6 +13,37 @@ namespace UnitsNet
     /// </summary>
     public class QuantityFormatter
     {
+        /// <summary>
+        /// The available UnitsNet custom format specifiers.
+        /// </summary>
+        private static readonly char[] UnitsNetFormatSpecifiers = { 'A', 'a', 'G', 'g', 'Q', 'q', 'S', 's', 'U', 'u', 'V', 'v' };
+
+        /// <summary>
+        /// Formats the given quantity using the given format string. Uses the <see cref="CultureInfo.CurrentUICulture" />.
+        /// </summary>
+        /// <typeparam name="TUnitType">The quantity's unit type, for example <see cref="LengthUnit"/>.</typeparam>
+        /// <param name="quantity">The quantity to format.</param>
+        /// <param name="format">The format string.</param>
+        /// <remarks>
+        /// The valid format strings are as follows:
+        /// Any of the standard numeric format strings for double values except for "G" or "g" ("C" or "c", "E" or "e", "F" or "f", "N" or "n", "P" or "p", "R" or "r").
+        /// "G" or "g": The value with 2 significant digits after the radix followed by the unit abbreviation, such as "1.23 m".
```

Comment:
Very nice docs, but it would probably not read so well in Visual Studio / Rider intellisense.

Try to add a list structure here, definition list probably works well for this:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/list

Reply:
I'll add that tomorrow!

Reply:
Added!